README.md

```diff
@@ -17,6 +17,21 @@ between two git revisions.
 
 It will configure `@owner` as the default owner in the config file.
 
+If you are just starting to use `codeowners-checker` in your project, you might not have clear ownership defined for all parts of the code.  In this case using the checker would be very tedious in the beginning, as it prompts for ownership rules for all files.
+
+You can still use `codeowners-checker` in this case without the tedium by providing a whitelist.  The whitelist contains patterns to ignore when checking files.
```

Comment:
```suggestion
You can still use `codeowners-checker` in this case without the tedium by providing a whitelist. The whitelist contains patterns to ignore when checking files.
```

Reply:
Old typing habits die hard!  I still do this to be able to navigate by sentence in Emacs and Vim :roll_eyes: 

Fixed it now